examples/linear_model/plot_ols_3d.py

```diff
@@ -41,7 +41,7 @@
 # Finally we plot the figure from three different views.
 
 import matplotlib.pyplot as plt
-
+from mpl_toolkits.mplot3d import Axes3D
```

Comment:
You would need to add `# noqa` for the linter to pass.
I propose to use the same import than in other examples:


```suggestion
# unused but required import for doing 3d projections with matplotlib < 3.2
import mpl_toolkits.mplot3d  # noqa: F401
```